remotes/errors/errdesc.go

```diff
@@ -100,7 +99,7 @@ func Register(group string, descriptor ErrorDescriptor) ErrorCode {
 	registerLock.Lock()
 	defer registerLock.Unlock()
 
-	descriptor.Code = ErrorCode(nextCode)
+	descriptor.Code = ErrorCode(descriptor.Value)
 
 	if _, ok := idToDescriptors[descriptor.Value]; ok {
```

Comment:
These maps seem redundant now since ErrorCode is also a string of the same value.

Reply:
yes, I have removed this map and some other unnecessary code. A new commit in this pr, you can see the changes that I have made. ^-^